### ISSUE TITLE
Navigate to each view when cycling through F6

### ISSUE THREAD

**Issue by eamodio** [microsoft/vscode#96480]
Refs: #96263

While testing this, I found myself wanting it to stop on each view in the side bar, so that you could navigate/interact with them quickly.

**Comment by isidorn** [comment]
This is a fair feature request, however not something which our screen reader users wanted. F6 should be a bird's eye navigation across the workbench and once on a section you can tab into it. You can easily navigate through each view by pressing tab and focusing inside the sidebar once the focus is on the sidebar.

However leaving this open as a feature request as more users might request this.
Thanks!
